modules/ting/ting.entities.inc

```diff
@@ -78,6 +86,26 @@ class TingEntity extends DingEntity {
    */
   public $reply;
```

Comment:
Can we change the visbility of the property now? I do not think we want to leave it as public.

Reply:
As I continue reading I can tell that unported modules mean we cannot change this yet...

Reply:
:+1: to the "not yet"